### DISCUSSION TITLE
Build of Hyperion.ng fails because of missing library -lc++abi

### DISCUSSION THREAD

**Discussion by thosoo** [termux/termux-packages#14718] [Q&A] [answered]
I am trying to build [Hyperion.ng](https://github.com/hyperion-project/hyperion.ng).
Installing dependencies and configuring using cmake worked fine.

However, I am getting the following error during build:
```
[ 13%] Linking CXX executable ../../../bin/flatc
ld.lld: error: unable to find library -lc++abi
clang-15: error: linker command failed with exit code 1 (use -v to see invocation)
make[2]: *** [dependencies/external/flatbuffers/CMakeFiles/flatc.dir/build.make:577: bin/flatc] Error 1
make[1]: *** [CMakeFiles/Makefile2:841: dependencies/external/flatbuffers/CMakeFiles/flatc.dir/all] Error 2
make: *** [Makefile:156: all] Error 2
```

I have figured that there is no libc++abi-devel like on linux distros to install, therefore I am quite stuck here.

**Comment by licy183** [comment, marked as answer]
Please install `ndk-multilib` and try again.

**Reply by thosoo** [reply]
Thanks! This helped with this issue 🎉
